libraries/AP_BattMonitor/AP_BattMonitor_TIBQ76952.cpp

```diff
@@ -0,0 +1,937 @@
+#include "AP_BattMonitor_config.h"
+
+#if AP_BATTERY_TIBQ76952_ENABLED
+
+#include <AP_HAL/AP_HAL.h>
+#include "AP_BattMonitor_TIBQ76952.h"
+#include "stdio.h"
+
+extern const AP_HAL::HAL& hal;
+
+#ifndef HAL_BATTMON_TIBQ76952_BUS
+#define HAL_BATTMON_TIBQ76952_BUS 0
+#endif
+#ifndef HAL_BATTMON_TIBQ76952_ADDR
+#define HAL_BATTMON_TIBQ76952_ADDR 0
```

Comment:
This doesn't look like a sensible default.

Shouldn't these be #error instead of creating defaults?  The hwdef should specify?

Reply:
the default is 0x08 it seems so I've changed this line to 0x08.  I've also removed the setting of the default from the VimDrones hwdef